### PR TITLE
set GC worker and loadDeleteRangeSQL high priority

### PR DESCRIPTION
**Pull request by coocood**
In case TiKV is busy, GC doesn't ever run.

### CODE REVIEW

**Comment by coocood** [comment]
@zhangjinpeng1987 PTAL

File: store/tikv/gcworker/gc_worker.go [hunk 1]
```diff
@@ -839,7 +839,7 @@ func (w *GCWorker) loadDurationWithDefault(key string, def time.Duration) (*time
 
 func (w *GCWorker) loadValueFromSysTable(key string, s session.Session) (string, error) {
 	ctx := context.Background()
-	stmt := fmt.Sprintf(`SELECT (variable_value) FROM mysql.tidb WHERE variable_name='%s' FOR UPDATE`, key)
+	stmt := fmt.Sprintf(`SELECT HIGH_PRIORITY (variable_value) FROM mysql.tidb WHERE variable_name='%s' FOR UPDATE`, key)
```
Comment (shenli) [inline store/tikv/gcworker/gc_worker.go]: Why there is a bracket around the `variable_value`?

Comment (coocood) [reply store/tikv/gcworker/gc_worker.go]: I think it doesn't matter.

Comment (shenli) [reply store/tikv/gcworker/gc_worker.go]: It looks weird but no harm.

**Comment by shenli** [comment]
LGTM

**Pull request merged**